hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip551/AtomicBatchInvalidSignaturesTests.java

```diff
@@ -0,0 +1,663 @@
+                    getTxnRecord("kycBatch").logged(),
+                    getTxnRecord(kyc1TxnId).assertingNothingAboutHashes().logged());
+        }
+    }
+
+    @Nested
+    @DisplayName("Token Update Batch Tests")
+    class TokenUpdateBatchTests {
+        private static final String SUPPLY_KEY = "supplyKey";
+        private static final String METADATA_KEY = "metadataKey";
+        private static final String NFT_TEST_METADATA = " test metadata";
+        private static final String RECEIVER = "receiver";
+
+        @HapiTest
+        @DisplayName("Batch fails when NFT metadata update lacks required signatures")
+        public Stream<DynamicTest> batchFailsWithoutMetadataKeySignature() {
+            final var batchOperator = "batchOperator";
+            final var nftToken = "nftToken";
+            final var updateTxnId = "updateTxnId";
+
+            return hapiTest(
+                    cryptoCreate(batchOperator).balance(ONE_HUNDRED_HBARS),
+                    newKeyNamed(SUPPLY_KEY),
+                    newKeyNamed(METADATA_KEY),
+                    cryptoCreate(TOKEN_TREASURY).balance(ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(4),
+                    cryptoCreate(RECEIVER).balance(ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(4),
+                    tokenCreate(nftToken)
+                            .supplyType(TokenSupplyType.FINITE)
+                            .tokenType(NON_FUNGIBLE_UNIQUE)
+                            .treasury(TOKEN_TREASURY)
+                            .maxSupply(12L)
+                            .supplyKey(SUPPLY_KEY)
+                            .metadataKey(METADATA_KEY)
+                            .initialSupply(0L),
+                    tokenAssociate(RECEIVER, nftToken),
+                    mintToken(nftToken, List.of(copyFromUtf8("a"), copyFromUtf8("b"))),
+                    // Verify initial setup
+                    getTokenNftInfo(nftToken, 1L).hasMetadata(copyFromUtf8("a")),
+
+                    // Batch with invalid NFT metadata update - should fail entire batch
+                    atomicBatch(
+                                    tokenUpdateNfts(nftToken, NFT_TEST_METADATA, List.of(1L))
+                                            .via(updateTxnId)
+                                            .batchKey(batchOperator)
+                                            .payingWith(TOKEN_TREASURY)
+                                            .fee(10 * ONE_HBAR) // Missing METADATA_KEY signature - INVALID_SIGNATURE
+                                    )
+                            .signedByPayerAnd(batchOperator)
+                            .via("metadataBatch")
+                            .hasKnownStatus(INNER_TRANSACTION_FAILED),
+
+                    // Verify batch failed and check transaction records
+                    getTxnRecord("metadataBatch").logged(),
+                    getTxnRecord(updateTxnId).assertingNothingAboutHashes().logged(),
```

Comment:
If I'm not mistaken, the streams validator at the end of each gradle test task already verifies the existence of all required records that were created by the node. In other words, it shouldn't be necessary to check that a record was generated–`getTxnRecord(...)` is only for doing a specific check on the record, so I believe we can get rid of these statements. (btw logging can significantly increase the time required for these tests to run, so we want to only use `.logged()` when necessary)